src/vos/evtree.c

```diff
@@ -34,8 +34,10 @@
 static inline void
 evt_ext_read(struct evt_extent *ext, const struct evt_rect_df *rin)
 {
+	uint64_t	len = ((uint64_t)rin->rd_len_hi << 16) + rin->rd_len_lo;
```

Comment:
maybe create a macro to do this for readability ... i.e. EVT_RECT_LEN(rin) ... though, I guess it's only one place.

Reply:
Better?